calculator/operations.py

```diff
@@ -1,23 +1,49 @@
 
 def add(*args):
-    # your implementation here
-    pass
+    output = 0
+    count = 0
+    for arg in args:
+        if count == 0:
+            output = arg
+        else:
+            output += arg
+        count += 1
+    return output
 
 def subtract(*args):
-    # your implementation here
-    pass
+    output = 0
+    count = 0
+    for arg in args:
```

Comment:
The same things apply from my comment on `add`. The only difference here is that you would set output initially to `args[0]` and your for loop to `for arg in args[1:]` (the [1:] returns the list from index 1 onwards)